app/models/miq_request.rb

```diff
@@ -166,6 +167,20 @@ def tracking_label_id
     "r#{id}_#{self.class.name.underscore}_#{id}"
   end
 
+  def load_transitions
+    self.state ||= 'pending'
+    {
+      :do_pending  => {'pending' => 'pending'},
+      :queued      => {'pending' => 'queued',
+                       'queued'  => 'queued',
```

Comment:
@hsong-rh why do we need the same state, if you are in queued you can stay in queued, it doesn't have to transition from queued to queued. In the method that validates the transition we can check the current state and next state and just ignore it

Reply:
I removed `'queued'  => 'queued'` and `'active' => 'active'`, but we still need `'pending' => 'pending'` in the beginning.